cmake/modules/AddCephTest.cmake

```diff
@@ -1,8 +1,30 @@
 #AddCephTest is a module for adding tests to the "make check" target which runs CTest
 
+macro(set_test_props labels cost cost_default timeout)
```

Comment:
i am inclined to avoid using macro to capture variables like `test_name`. it's difficult to maintain and hurts the readability.

Reply:
could you pass the `timeout_default` explicitly as well?

Reply:
@tchaikov 
In this macro I make explicit use of the fact that a macro does replacement on text level, instead of programmatic functionality.
It is perhaps a bit less obvious, but I explicitly chose this, since otherwise the other routines become even less ledgible. So I prefer to keep this, perhaps a bit better documented.

Reply:
@tchaikov 
Since parameters are positional here, passing `timeout_default` here requires also setting something for `timeout`. And this then surpasses the tact that CEPH_TEST_TIMEOUT is a globally set value already. This is an escapehatch for tests that need more than the alloted testtime, manually overriden if needed.